.github/workflows/api-db-audit.yml

```
@@ -0,0 +1,89 @@
+    - name: Install Dependencies
+      run: |
+        pip install requests
+        dotnet restore Backend/BARQ.sln
+    
+    - name: Build & apply EF migrations
+      run: |
+        dotnet tool install --global dotnet-ef || true
+        echo "$HOME/.dotnet/tools" >> $GITHUB_PATH
+        dotnet build Backend/BARQ.sln -c Release
+        dotnet ef database update --project Backend/src/BARQ.Infrastructure --startup-project Backend/src/BARQ.API
+    
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)